language/func-api.md

```diff
@@ -37,6 +37,9 @@ It is not uncommon that 80% of the logic in  function consists of argument type
 is when there is no checking at all (because it is a chore to write) leading to mysterious
 and sometimes spectacular failures.
 
+For autoloading information see [Autoloading][1].
```

Comment:
this don't render right it should be a lowercase in the link and uppercase in text, maybe try `[Autoloading](#autoloading)`?